tests/core/engine_adapter/test_integration.py

```diff
@@ -2208,6 +2235,15 @@ def test_to_time_column(
     if ctx.test_type != "query":
         pytest.skip("Time column tests only need to run for query")
 
+    if ctx.dialect == "clickhouse" and time_column_type.is_type(exp.DataType.Type.TIMESTAMPTZ):
+        # Clickhouse does not have natively timezone-aware types and does not accept timestrings
+        #   with UTC offset "+XX:XX". Therefore, we remove the timezone offset and set a timezone-
+        #   specific data type to validate what is returned.
+        import re
+
+        time_column = re.match("^(.*?)\+", time_column).group(1)
+        time_column_type = exp.DataType.build("TIMESTAMP('UTC')", dialect="clickhouse")
```

Comment:
@erindru will we still need this after we merge this change https://github.com/tobymao/sqlglot/pull/3938/files#diff-f2821f7be91ef6edbb6ed1683c3fa2ef31f23201961ba7757a4c37687f2ad896R120-R126? If not, let's leave a TODO to clean it up later.